### DISCUSSION TITLE
Generate links automatically

### DISCUSSION THREAD

**Discussion by octvs** [jakewvincent/mkdnflow.nvim#159] [General]
Hey there,
Is there a way to create links to all other (probable) note files in child directories using Mkdnflow. There was a :VimwikiGenerateLinks cmd in vimwiki, which accomplished most of what I'm looking for. Bonus if it can turn sub-dirs to headers but I'm not sure if that is feasible since the depth of the files can be arbitrary.
At the end it will create a unordered index file.

**Comment by jrop** [comment]
Came here looking for the same thing. This is the only thing keeping me from switching away from Vimwiki to mkdnflow. I keep hoping that I'll get some time to write this myself, but I've been swamped lately. All in all, I'm stoked that yet one more plugin in my config will be Lua instead of Vimscript (soon).

Also, now that I think about it, tag search would be awesome to have too. I wonder if each of these concerns should be rolled up into this plugin, or separate plugins.

**Comment by jakewvincent** [comment]
Created an issue from this here: https://github.com/jakewvincent/mkdnflow.nvim/issues/222
